tests/Aspire.Hosting.Oracle.Tests/OracleFunctionalTests.cs

```diff
@@ -0,0 +1,370 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Components.Common.Tests;
+using Aspire.Hosting.Tests.Utils;
+using Aspire.Hosting.Utils;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Polly;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Aspire.Hosting.Oracle.Tests;
+
+public class OracleFunctionalTests(ITestOutputHelper testOutputHelper)
+{
+    // Folders created for mounted folders need to be granted specific permissions
+    // for the non-root user in the container to be able to access them.
+
+    private const UnixFileMode MountFilePermissions =
+       UnixFileMode.UserRead | UnixFileMode.UserWrite | UnixFileMode.UserExecute |
+       UnixFileMode.OtherRead | UnixFileMode.OtherWrite | UnixFileMode.OtherExecute;
+
+    private const string DatabaseReadyText = "Completed: ALTER DATABASE OPEN";
```

Comment:
We should use the const everywhere. It is currently only used in 1 spot and the string is duplicated across tests.